spec/features/saml/aal3_required_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'AAL3 authentication required in an SAML context' do
+  include SamlAuthHelper
+
+  describe 'SAML ServiceProvider requesting AAL3 authentication' do
+    pending 'Pending actual support of multiple auth contexts in SAML'
+
+    context 'user does not have aal3 auth configured' do
+      it 'sends user to set up AAL3 auth'
```

Comment:
you can still leave the test bodies in here! pending spec failures don't count (they're actually supposed to fail, otherwise it yells at you to take off "pending")

Reply:
but that's neither here nor there, no need to fix that again before merging

Reply:
The test bodies were copied/pasted from the oidc version of the tests, so really, they shouldn't have been here anyway. `pending` doesn't prevent the code from being run, and it was failing since I was calling methods from the oidc helpers.

Reply:
oh got it!